.github/workflows/ci.yml

```diff
@@ -63,6 +72,10 @@ jobs:
         if: runner.os == 'Windows'
         run: choco install psql -y --no-progress
         shell: powershell
+      - name: Start Docker daemon
+        if: runner.os == 'Linux'
+        run: /usr/local/bin/start-docker.sh
```

Comment:
We could simplify this a lot by just not running a docker container for the smoketest. I think it would probably be fine to use `spacetime start` here instead. We could even just start one instance of `spacetime start` per test for running them in parallel

Reply:
we use a docker container so that we can test the docker based tests, e.g. restarting spacetimedb